src/app/zap-templates/templates/app/cluster-enums.zapt

```diff
@@ -24,7 +24,7 @@ enum class {{asType label}} : {{asUnderlyingZclType name}} {
 {{#zcl_enum_items}}
 k{{asUpperCamelCase label}} = {{asHex value 2}},
 {{/zcl_enum_items}}
-// kUnknownEnumValue = {{first_unused_enum_value mode="first_unused"}},
+kUnknownEnumValue = {{first_unused_enum_value mode="first_unused"}},
```

Comment:
All the other enum values are in hex. Is there an easy way to make it hex also, just for consistency?